src/main/resources/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "projectID": "your project ID"
```

Comment:
Is it okay to use "your project ID" here?

Reply:
yes, it's just default file, so you won't need to create file and folder for it manually, you need only change "your project ID" with project id

Reply:
I can change it to `""` and add a comment btw, but I don't think that it's equivalent

Reply:
Ok, I just wanted to make sure